core.impl/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/impl/repair/RepairGroup.java

```diff
@@ -67,6 +82,20 @@ public RepairGroup(final int priority, final Builder builder)
                 .checkNotNull(builder.myTableRepairMetrics, "Table repair metrics must be set");
         myRepairPolicies = new ArrayList<>(Preconditions
                 .checkNotNull(builder.myRepairPolicies, "Repair policies must be set"));
+        if (!myRepairConfiguration.getRepairType().equals(RepairType.INCREMENTAL))
```

Comment:
F: call the equals on the enum and you will never need to care about NullPointerException

RepairType.INCREMENTAL.equals(myRepairConfiguration.getRepairType())